GliaWidgetsTests/Sources/CallViewModelTests.swift

```diff
@@ -335,4 +335,29 @@ class CallViewModelTests: XCTestCase {
 
         XCTAssertEqual(call.kind.value, .audio)
     }
+
+    func test_liveObservationAllowed() throws {
+        let viewModel: CallViewModel = .mock()
+        let alertConfirmation: CallViewController.LiveObservationConfirmation = .init(
+            conf: .mock(),
+            accepted: {
+                viewModel.interactor.state = .enqueued(.mock)
+            },
+            declined: {}
+        )
+
+        alertConfirmation.accepted()
+        XCTAssertEqual(viewModel.interactor.state, .enqueued(.mock))
```

Comment:
I think these test make no sense, because you test updating from the button configuration closure. 
Would be beneficial to test:
- `CallViewModel.start()` does starts an engagement
- `EngagementViewModel.engagementAction?(.showLiveObservationConfirmation(liveObservationAlertConfig))` is called when Interactor state is changed to appropriate value
- `interactor.enqueueForEngagement` is called when Accept action of the LiveObservationConfirmation alert is called
- `interactor.endSession` is called or appropriate interactor state is chenged when Decline action of the LiveObservationConfirmation alert is called

Reply:
fixed [95da2c2](https://github.com/salemove/ios-sdk-widgets/pull/787/commits/95da2c2d67e8fd0b3fddcd08b23139fe16fe4026)